src/EPR.Accreditation.Facade/Services/AccreditationService.cs

```diff
@@ -275,26 +275,28 @@ private Common.Dtos.Accreditation CreateNewAccreditationDto(OperatorType operato
             Guid overseasSiteExternalId,
             OverseasReprocessingSiteOutputs overseasSiteOutputs)
         {
-            var overseasSite = new OverseasReprocessingSite { 
-                ExternalId = overseasSiteExternalId, 
-                Outputs = overseasSiteOutputs.Outputs };
+            var overseasSite = new OverseasReprocessingSite
+            {
+                ExternalId = overseasSiteExternalId,
+                Outputs = overseasSiteOutputs.Outputs
+            };
             await _httpAccreditationService.UpdateOverseasReprocessingSite(accreditationExternalId, overseasSite);
         }
 
         public async Task<HasOverseasAgentDto> GetHasOverseasAgent(Guid accreditationExternalId)
         {
             var accreditation = await _httpAccreditationService.GetAccreditation(accreditationExternalId);
 
-            return accreditation == null 
-                ? new HasOverseasAgentDto() 
-                : new HasOverseasAgentDto 
-                    { 
-                        HasOverseasAgent = accreditation.HasOverseasAgent
-                    };
+            return accreditation == null
```

Comment:
Um.... if accreditation is null surely it doesn't exist? Therefore why are you returning valid objects? Surely a not found is the currect response?